metricbeat/module/elasticsearch/cluster_stats/_meta/data.json

```diff
@@ -1,43 +1,451 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
-    "elasticsearch": {
-        "cluster": {
-            "id": "6UTQ_iuNSzWP49zv99vxDg",
-            "name": "elasticsearch",
-            "stats": {
-                "indices": {
-                    "fielddata": {
-                        "memory": {
-                            "bytes": 1208
-                        }
-                    },
-                    "shards": {
-                        "count": 18,
-                        "primaries": 18
-                    },
-                    "total": 18
```

Comment:
Looks like you changed this to `count`, which would be a breaking change. I would suggest reverting this to `total` again.

In general, we should not introduce breaking changes since this PR will be eventually backported to `7.x`. Its okay to introduce new fields in this PR but not okay to take away existing fields. To ensure this, I would suggest comparing the old `data.json` with the new `data.json` using something like jsondiff.com and making sure the changes from old => new are purely additive.

Reply:
Good catch! Thank you! I'll start using jsondiff, it will definitely help me checking backwards compatibility.